### PR TITLE
add upgrading docs

### PR DESCRIPTION
**Pull request by nslater**
pr for the new upgrading docs

### CODE REVIEW

**Comment by nslater** [comment]
replaces #5016

File: blackbox/docs/migrating.txt [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+Migrating to 1.1
+================
+
+CrateDB 1.1 introduces a new internal table format that is backwards
+incompatible with previous table formats. Future versions of CrateDB will only
+support this new table format. CrateDB 1.1 will be the only release that can
+read all older table formats *and* the new table format.
+
+CrateDB 1.1 includes new cluster checks that will tell you which tables need
```
Comment (matriv) [inline blackbox/docs/migrating.txt]: maybe tell you -> inform you of

File: blackbox/docs/migrating.txt [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+.. figure:: _static/upgrade-cluster-checks.png
+   :align: center
+
+   *Figure 1*: Both cluster checks are failing in this example.
+
+If your cluster checks are green, there is nothing for you to do. Otherwise,
+continue to the section `Preparation`_.
+
+Preparation
```
Comment (matriv) [inline blackbox/docs/migrating.txt]: maybe we need to skip this step as the upgrade is considered safe and for the recreate you export your schema and data anyways. @seut right?

File: blackbox/docs/migrating.txt [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+If you exported your table like in the previous example, you could then import
+your data one file at a time, like so::
+
+  cr> COPY table_ident FROM '/data/backups/table_ident_0_.json';
+
+See the :ref:`copy_from` command documentation for more information.
+
+When all tables that needed recreating have been exported, dropped, recreated,
+and reimported, you can move on.
```
Comment (matriv) [inline blackbox/docs/migrating.txt]: maybe also add the the relevant cluster check should disappear after that.

Comment (nslater) [inline blackbox/docs/migrating.txt]: yeah I think it's fine as is

File: blackbox/docs/migrating.txt (hunk 2 above)
Comment (nslater) [inline blackbox/docs/migrating.txt]: okay. dropping

File: blackbox/docs/migrating.txt (hunk 1 above)
Comment (nslater) [inline blackbox/docs/migrating.txt]: done

**Pull request closed without merging**